ai/STA/Strategy/strategy_book.py

```diff
@@ -80,10 +90,11 @@ def get_strategy(self, strategy_name: str) -> Strategy:
         :param strategy_name: (str) le nom de la stratégie à retourner
         :return: (Tactic) une nouvelle instance de la stratégie demandé.
         """
+        assert isinstance(strategy_name, str)
```

Comment:
Typing should be enough.

Reply:
In this case, a mistake in the typing caused problems. Now it's fix, but I want to be sure that this part is respected in the future.